simulariumio/__init__.py

```diff
@@ -22,9 +22,13 @@
     ScatterPlotData,
     TrajectoryData,
     UnitData,
+    FrameData,
+    SimulariumFileData,
+    JsonData,
+    BinaryData,
 )
 # DO NOT ISORT DISPLAY_TYPE, CAUSES CIRCULAR DEP
-from .constants import DISPLAY_TYPE  # noqa: F401
+from .constants import BINARY_SETTINGS, DISPLAY_TYPE  # noqa: F401
```

Comment:
this looks like an implementation detail and makes me wonder if BINARY_SETTINGS should live inside of BinaryData.  Just seeing this in isolation so not sure if that would make sense